packages/sanity/src/config/types.ts

```diff
@@ -9,7 +9,7 @@ import type {
   SchemaTypeDefinition,
 } from '@sanity/types'
 import type React from 'react'
-import {ComponentType, ReactNode} from 'react'
+import type {ComponentType, ReactNode} from 'react'
```

Comment:
Just curious: is this change necessary? If so, there are a lot of places to update?

Reply:
@rexxars suggested this in his review:
https://github.com/sanity-io/sanity/pull/3612#discussion_r971501519
![image](https://user-images.githubusercontent.com/81981/193017689-31a4338f-2a6d-48bf-b480-68bb3a395356.png)

I don't have an opinion here and will follow the consensus 😄